interpreter/shared/src/main/scala/org/ergoplatform/ErgoBox.scala

```diff
@@ -103,10 +103,13 @@ object ErgoBox {
     val size: Short = 32
   }
 
-  type TokenId = Digest32
+  /** Token id is tagged collection of bytes. */
+  type TokenId = Digest32Coll
```

Comment:
is this change really needed? would break a lot of things upstream

If needed, make alt constructor with old arguments at least

Reply:
Alt constructor of ErgoBox. You can modify Ergo node and Appkit, but what about Kiosk and other tools and apps?

Reply:
Kiosk is using Appkit and the other apps probably too. 
Sigma has always been discouraged to be used directly (see second paragraph in readme https://github.com/ScorexFoundation/sigmastate-interpreter#ergoscript-compiler-and-ergotree-interpreter).

This is not the first breaking change and also not the most significant and has not been any complains so far.
The most significant was switch to v5.0 which was smooth because Sigma changes was properly hidden behind Appkit's abstractions.
So applications which use direct dependencies on Sigma will have to be updated.

Once refactoring is done (hopefully soon), we can do stabilization of sigma APIs in sigma `sdk` module.
My plan is to make `sdk` module a stable abstraction layer for Scala applications so that using Appkit will not be necessary from Scala projects.
Appkit itself will become a thin wrapper over sigma sdk, so most of the logic will migrate from Appkit to Sigma SDK. (AppkitProvingInterpreter is one example of it, and Extended keys is another).